yt_dlp/jsinterp.py

```diff
@@ -280,6 +282,10 @@ def _operator(self, op, left_val, right_expr, expr, local_vars, allow_recursion)
             return right_val
 
         try:
+            # https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators#binary_bitwise_operators
+            if op in ('&', '|', '^', '~', '<<', '>>', '>>', '>>>'):
+                return c_int32(_OPERATORS[op](left_val, right_val)).value
```

Comment:
I believe this does work without ctypes
```suggestion
                return _OPERATORS[op](left_val, right_val) & 0xffffffff
```

Reply:
It does 😮